.editorconfig

```diff
@@ -12,10 +12,10 @@ trim_trailing_whitespace = true
 trim_trailing_whitespace = false
 
 [*.yml]
-indent_size = 2
+indent_size = 4
 
 [*.js]
-indent_size = 2
+indent_size = 4
 
 [*.vue]
-indent_size = 2
+indent_size = 4
```

Comment:
@stepanenko3 can yo not change these configs please?

Reply:
@bkintanar Yes, of course

Reply:
@stepanenko3 and kindly revert the changes made by this change for the vue files. there are a lot of modifications in vue files that are related to indention.

Reply:
@bkintanar, I don't have time to change everything right now

But I mark as checked "Allow edits by maintainers" if you have time for this

Reply:
@bkintanar I found a couple of minutes for this, and fixed it in https://github.com/ebess/advanced-nova-media-library/pull/325/commits/9a5580fa974d29bfb11a11ba79a3397a02a3e8ff